src/api/fship_app/views.py

```diff
@@ -0,0 +1,387 @@
+            name=record["name"], 
+            gitHandle=record["github"], 
+            email=record["email"]
+        )
+
+    ## Close the session
+    session.close()
+
+    return HttpResponse(json.dumps(person.toJSON()), content_type='application/json; charset=UTF-8')
+
+## Get a user's hobbies
+def getUserHobies(request, id):
+    ## Start the session
+    session = driver.session()
+    # Define the query
+    query = f"""
+    MATCH (u:User) - [:hasHobbie] -> (h:Hobbie) WHERE id(u) = {id} 
+    RETURN id(h) AS id, h.name AS hobby
+    """
+
+    result = session.run(query) ## Execute the function 
+    ## The result is not subsciptible so we loop to get the single value 
+    ## Get the user details
+    hobbies = [
+        models.Hobby(
+            id=record["id"], 
+            name=record["hobby"], 
+        ).toJSON()
+        for record in result
+    ]
+    ## Close the session
+    session.close()
+
+    ## Create a dict with id and hobbies
+    response = {
+        "user_id" : id,
+        "hobbies" : hobbies,
+    }
+
+    return HttpResponse(json.dumps(response), content_type='application/json; charset=UTF-8')
+
+
+def getUserSkills(request, id):
+    ## Start the session
+    session = driver.session()
+    # Define the query
+    query = f"""
+    MATCH (u:User) - [:isInterestedIn] -> (s:TechSkill) WHERE id(u) = {id} 
+    RETURN id(s) AS id, s.name AS skill
+    """ 
+
+    result = session.run(query) ## Execute the function 
+    ## The result is not subsciptible so we loop to get the single value 
+    ## Get the user details
+    skills = [
+        models.TechSkill(
+            id=record["id"], 
+            name=record["skill"], 
+        ).toJSON()
+        for record in result
+    ]
+    ## Close the session
+    session.close()
+
+    ## Create a dict with id and hobbies
+    response = {
+        "user_id" : id,
+        "skills" : skills,
+    }
+
+    return HttpResponse(json.dumps(response), content_type='application/json; charset=UTF-8')
+
+
+# Function to get a user's bio
+def getUserBio(request, id):
+    ## Start the session
+    session = driver.session()
+    # Define the query
+    query = f"""
+    MATCH (u:User) - [:hasBio] -> (b:Bio) WHERE id(u) = {id} RETURN id(b) AS id, 
+    b.description AS description
+    """ 
+
+    result = session.run(query) ## Execute the function 
+    ## The result is not subsciptible so we loop to get the single value 
+    ## Get the user details
+    for record in result:
+        bio = models.Bio(
+            id=record["id"], 
+            description=record["description"], 
+        ).toJSON()
+
+    ## Close the session
+    session.close()
+
+    ## Create a dict with id and hobbies
+    response = {
+        "user_id" : id,
+        "bio" : bio,
+    }
+
+    return HttpResponse(json.dumps(response), content_type='application/json; charset=UTF-8')
+
+
+## Function to get a user's country and timeZone
+def getUserTimeZoneAndCountry(request, id):
```

Comment:
This endpoint is giving me an error. Please make sure to fix it.

Reply:
This issue has been resolved on the #9 PR. You can add the sample data found on ReadMe of the same PR and test it out.